src/chart/funnel/funnelLayout.js

```diff
@@ -104,8 +104,42 @@ function labelLayout(data) {
                 x2 = x1 - labelLineLen;
                 textX = x2 - 5;
                 textAlign = 'right';
-            }
-            else {
+            } else if (labelPosition === 'right') {
```

Comment:
There should be a new line before `else`.

Reply:
yeah，I need to update the code.